test/spy-test.js

```diff
@@ -2226,6 +2226,34 @@
 
                 assert.equals(spy.length, 3);
             }
+        },
+
+        "etters": {
```

Comment:
I don't like the naming here. Either `getters and setters` or a `getters` section separate from the `setters` section.